commands/addvertical.js

```diff
@@ -84,10 +84,20 @@ class VerticalAdder {
     if (!defaultTheme || !themesDir) {
       return [];
     }
-    const cardsDir = path.join(themesDir, defaultTheme, 'cards');
-    return fs.readdirSync(cardsDir, { withFileTypes: true })
+    const themeCardsDir = path.join(themesDir, defaultTheme, 'cards');
+
+    const cards = fs.readdirSync(themeCardsDir, { withFileTypes: true })
       .filter(dirent => !dirent.isFile())
       .map(dirent => dirent.name);
+
+    const customCardsDir = 'cards';
+    if (fs.existsSync(customCardsDir)) {
```

Comment:
will this have duplicate cards if somebody overrides a theme one? like if somebody overrides faq-accordion will there be two faq-accordions in the array?

Reply:
I believe we prevent people from naming their custom card the same as a built-in one. I just tried that and the `card` command threw an error saying the name was taken.

Reply:
ah I see. I think people can still shadow the card though? (I'm just remembering mercy overriding link-standard and faq-accordion but only the component.js parts)

Reply:
Still might be worth just having this return a set instead? Just on the off chance of something weird

Reply:
Yeah, I suppose people could do that. Although, they really should just make a new card in that case. But, I can update to handle!